web/packages/teleport/src/AppLauncher/AppLauncher.tsx

```diff
@@ -140,6 +146,19 @@ export function AppLauncherAccessDenied(props: AppLauncherAccessDeniedProps) {
   return <AccessDenied message={props.statusText} />;
 }
 
+// stripFQDN removes the port from the FQDN if it has one and ensures
+// the FQDN is lowercase. This is to prevent issues matching the
+// resolved fqdn with the one that was passed. Apps generally aren't
+// supposed to have a port in the public address but some integrations
+// create apps that do.
+function stripFQDN(fqdn: string) {
```

Comment:
could we use maybe [new URL](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL)? and we can get the hostname without the port that way.

i don't think the fqdn returns with the protocol (but then again, it's not supposed to include the port either...), so with missing protocol, we can just add one `https://${fqdn}`

Reply:
For some reason that doesn't work... when you create a URL object from a URL string with a port, the returned URL object's `port` property is an empty string for some reason. So setting it to an empty string to try and remove the port has no effect.

Reply:
??

<img width="318" alt="image" src="https://github.com/user-attachments/assets/af8d379c-1003-4274-a003-6e43baa1bbc6">

Reply:
Ok apparently there's different behaviors depending on the JS runtime, what works in Node does not work in Firefox/chromium... I originally set the scheme to `scheme` so if the prepared URLs used to match were accidentally redirected to it wouldn't work. With Node this works fine, but with chromium/Firefox if the scheme isn't recognized the port is set to an empty string

Reply:
The current commit works, I just opted to use `https` as the scheme as @kimlisa originally suggested